vscode/src/extension.ts

```diff
@@ -184,6 +184,36 @@ async function activateLanguageService(extensionUri: vscode.Uri) {
     ...registerQSharpNotebookCellUpdateHandlers(languageService),
   );
 
+  // Watch manifest changes and update each document in the same project as the manifest.
+  subscriptions.push(
```

Comment:
Can this code live inside `registerDocumentUpdateHandlers`? That function is meant to group together all the language service synchronization / update behavior for Q# documents. I think this logically belongs there.

Reply:
Yes, you are right. I just moved it to `registerDocumentUpdateHandlers`.